fiaas_mast/common.py

```diff
@@ -53,10 +52,9 @@ def make_safe_name(name):
 
 
 def select_models():
-    for app_model, spec_model in (
-            (FiaasApplication, FiaasApplicationSpec),
-            (PaasbetaApplication, PaasbetaApplicationSpec),
-    ):
+    for app_model, spec_model in [
+            (FiaasApplication, FiaasApplicationSpec)
```

Comment:
Nit: the error below still refers to the PaasbetaApplication.

Reply:
Addressed in https://github.com/fiaas/mast/pull/26/commits/78410a8b345ed1572bd4ceda1faad10890cd403b